schema-registry/rest-service/src/test/java/com/hortonworks/registries/schemaregistry/avro/SchemaBranchLifeCycleTest.java

```diff
@@ -340,14 +345,18 @@ public void deleteInvalidSchemaBranch() throws IOException, SchemaBranchNotFound
 
     @Test (expected = InvalidSchemaBranchDeletionException.class)
     public void deleteMasterBranch() throws IOException, SchemaBranchNotFoundException, InvalidSchemaException, SchemaNotFoundException, IncompatibleSchemaException, SchemaBranchAlreadyExistsException, InvalidSchemaBranchDeletionException {
+        SchemaMetadata schemaMetadata = createSchemaMetadata(testNameRule.getMethodName(), SchemaCompatibility.NONE);
```

Comment:
Do you really need to add this here?

Reply:
@raju-saravanan clarified that it is needed as the branch created when a schema metadata is created for local registry server used in the test.